### PR TITLE
Skin render program image processing

### PR DESCRIPTION
**Pull request by LarsIngo**

### CODE REVIEW

File: src/Video/VideoErrorCheck.cpp [hunk 1]
```diff
@@ -0,0 +1,27 @@
+#include "VideoErrorCheck.hpp"
+
+#include <GL/glew.h>
+#include <Utility/Log.hpp>
+
+VideoErrorCheck::VideoErrorCheck(const std::string& name) {
+    this->name = name;
```
Comment (NinjaDanz3r) [inline src/Video/VideoErrorCheck.cpp]: MINOR Variable 'name' is assigned in constructor body. Consider performing initialization in initialization list. [rule](http://localhost:9000/coding_rules#rule_key=cppcheck%3AuseInitializationList)

**Comment by NinjaDanz3r** [comment]
SonarQube analysis reported 9 issues
* ![MINOR][MINOR] 9 minor

Watch the comments in this conversation to review them.

#### 8 extra issues

Note: The following issues were found on lines that were not modified in the pull request. Because these issues can't be reported as line comments, they are summarized here:

1. ![MINOR][MINOR] AncientFungus:LGEPR: Cppcheck cannot find all the include files (use --check-config for details) [rule](http://localhost:9000/coding_rules#rule_key=cppcheck%3AmissingInclude)
1. ![MINOR][MINOR] [RenderManager.cpp#L507](https://github.com/Chainsawkitten/LargeGameProjectEngine/blob/84e51a11e5b5f084d99f7d1e30832d314b8e5ae0/src/Engine/Manager/RenderManager.cpp#L507): The function 'GetAnimations' is never used. [rule](http://localhost:9000/coding_rules#rule_key=cppcheck%3AunusedFunction)
1. ![MINOR][MINOR] [RenderManager.cpp#L525](https://github.com/Chainsawkitten/LargeGameProjectEngine/blob/84e51a11e5b5f084d99f7d1e30832d314b8e5ae0/src/Engine/Manager/RenderManager.cpp#L525): The function 'GetDirectionalLights' is never used. [rule](http://localhost:9000/coding_rules#rule_key=cppcheck%3AunusedFunction)
1. ![MINOR][MINOR] [RenderManager.cpp#L544](https://github.com/Chainsawkitten/LargeGameProjectEngine/blob/84e51a11e5b5f084d99f7d1e30832d314b8e5ae0/src/Engine/Manager/RenderManager.cpp#L544): The function 'GetLenses' is never used. [rule](http://localhost:9000/coding_rules#rule_key=cppcheck%3AunusedFunction)
1. ![MINOR][MINOR] [RenderManager.cpp#L564](https://github.com/Chainsawkitten/LargeGameProjectEngine/blob/84e51a11e5b5f084d99f7d1e30832d314b8e5ae0/src/Engine/Manager/RenderManager.cpp#L564): The function 'GetMaterials' is never used. [rule](http://localhost:9000/coding_rules#rule_key=cppcheck%3AunusedFunction)
1. ![MINOR][MINOR] [RenderManager.cpp#L582](https://github.com/Chainsawkitten/LargeGameProjectEngine/blob/84e51a11e5b5f084d99f7d1e30832d314b8e5ae0/src/Engine/Manager/RenderManager.cpp#L582): The function 'GetMeshes' is never used. [rule](http://localhost:9000/coding_rules#rule_key=cppcheck%3AunusedFunction)
1. ![MINOR][MINOR] [RenderManager.cpp#L601](https://github.com/Chainsawkitten/LargeGameProjectEngine/blob/84e51a11e5b5f084d99f7d1e30832d314b8e5ae0/src/Engine/Manager/RenderManager.cpp#L601): The function 'GetPointLights' is never used. [rule](http://localhost:9000/coding_rules#rule_key=cppcheck%3AunusedFunction)
1. ![MINOR][MINOR] [RenderManager.cpp#L623](https://github.com/Chainsawkitten/LargeGameProjectEngine/blob/84e51a11e5b5f084d99f7d1e30832d314b8e5ae0/src/Engine/Manager/RenderManager.cpp#L623): The function 'GetSpotLights' is never used. [rule](http://localhost:9000/coding_rules#rule_key=cppcheck%3AunusedFunction)

[MINOR]: https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR'

**Comment by NinjaDanz3r** [comment]
SonarQube analysis reported 9 issues
* ![MINOR][MINOR] 9 minor

Watch the comments in this conversation to review them.

#### 8 extra issues

Note: The following issues were found on lines that were not modified in the pull request. Because these issues can't be reported as line comments, they are summarized here:

1. ![MINOR][MINOR] AncientFungus:LGEPR: Cppcheck cannot find all the include files (use --check-config for details) [rule](http://localhost:9000/coding_rules#rule_key=cppcheck%3AmissingInclude)
1. ![MINOR][MINOR] [RenderManager.cpp#L507](https://github.com/Chainsawkitten/LargeGameProjectEngine/blob/810d424424af576aa392cde65cdd58d3386cf913/src/Engine/Manager/RenderManager.cpp#L507): The function 'GetAnimations' is never used. [rule](http://localhost:9000/coding_rules#rule_key=cppcheck%3AunusedFunction)
1. ![MINOR][MINOR] [RenderManager.cpp#L525](https://github.com/Chainsawkitten/LargeGameProjectEngine/blob/810d424424af576aa392cde65cdd58d3386cf913/src/Engine/Manager/RenderManager.cpp#L525): The function 'GetDirectionalLights' is never used. [rule](http://localhost:9000/coding_rules#rule_key=cppcheck%3AunusedFunction)
1. ![MINOR][MINOR] [RenderManager.cpp#L544](https://github.com/Chainsawkitten/LargeGameProjectEngine/blob/810d424424af576aa392cde65cdd58d3386cf913/src/Engine/Manager/RenderManager.cpp#L544): The function 'GetLenses' is never used. [rule](http://localhost:9000/coding_rules#rule_key=cppcheck%3AunusedFunction)
1. ![MINOR][MINOR] [RenderManager.cpp#L564](https://github.com/Chainsawkitten/LargeGameProjectEngine/blob/810d424424af576aa392cde65cdd58d3386cf913/src/Engine/Manager/RenderManager.cpp#L564): The function 'GetMaterials' is never used. [rule](http://localhost:9000/coding_rules#rule_key=cppcheck%3AunusedFunction)
1. ![MINOR][MINOR] [RenderManager.cpp#L582](https://github.com/Chainsawkitten/LargeGameProjectEngine/blob/810d424424af576aa392cde65cdd58d3386cf913/src/Engine/Manager/RenderManager.cpp#L582): The function 'GetMeshes' is never used. [rule](http://localhost:9000/coding_rules#rule_key=cppcheck%3AunusedFunction)
1. ![MINOR][MINOR] [RenderManager.cpp#L601](https://github.com/Chainsawkitten/LargeGameProjectEngine/blob/810d424424af576aa392cde65cdd58d3386cf913/src/Engine/Manager/RenderManager.cpp#L601): The function 'GetPointLights' is never used. [rule](http://localhost:9000/coding_rules#rule_key=cppcheck%3AunusedFunction)
1. ![MINOR][MINOR] [RenderManager.cpp#L623](https://github.com/Chainsawkitten/LargeGameProjectEngine/blob/810d424424af576aa392cde65cdd58d3386cf913/src/Engine/Manager/RenderManager.cpp#L623): The function 'GetSpotLights' is never used. [rule](http://localhost:9000/coding_rules#rule_key=cppcheck%3AunusedFunction)

[MINOR]: https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR'

File: src/Engine/Manager/RenderManager.cpp [hunk 2]
```diff
@@ -257,138 +259,149 @@ void RenderManager::RenderWorldEntities(World& world, const glm::mat4& viewMatri
...
-        LightWorld(world, viewMatrix, projectionMatrix, viewProjectionMatrix);
+    // Lights.
+    { VIDEO_ERROR_CHECK("Update lights");
+    { PROFILE("Update lights");
+    { GPUPROFILE("Update lights", Video::Query::Type::TIME_ELAPSED);
+        if (lighting) {
+            // Cull lights and update light list.
+            LightWorld(world, viewMatrix, projectionMatrix, viewProjectionMatrix);
+        }
+        else {
```
Comment (Chainsawkitten) [inline src/Engine/Manager/RenderManager.cpp]: else should be on same line as bracket

File: src/Video/Buffer/StorageBuffer.cpp [hunk 3]
```diff
@@ -3,8 +3,8 @@
 #include <assert.h>
 #include <cstring>
 
+#include "VideoErrorCheck.hpp"
```
Comment (Chainsawkitten) [inline src/Video/Buffer/StorageBuffer.cpp]: When using relative include, actually use relative include path: `"../VideoErrorCheck.hpp"`

File: src/Video/RenderProgram/SkinRenderProgram.cpp [hunk 4]
```diff
@@ -20,10 +23,78 @@ SkinRenderProgram::SkinRenderProgram() {
...
     delete vertexShader;
     delete fragmentShader;
+
+    //Create shaders for early rejection pass
+    vertexShader = new Shader(ZREJECTIONSKIN_VERT, ZREJECTIONSKIN_VERT_LENGTH, GL_VERTEX_SHADER);
+    fragmentShader = new Shader(ZREJECTION_FRAG, ZREJECTION_FRAG_LENGTH, GL_FRAGMENT_SHADER);
+    zShaderProgram = new ShaderProgram({ vertexShader, fragmentShader });
+    delete vertexShader;
+
+    ////Create shaders for shadowpass
```
Comment (Chainsawkitten) [inline src/Video/RenderProgram/SkinRenderProgram.cpp]: Why four slahes?

**Comment by NinjaDanz3r** [comment]
SonarQube analysis reported 9 issues
* ![MINOR][MINOR] 9 minor

Watch the comments in this conversation to review them.

#### 8 extra issues

Note: The following issues were found on lines that were not modified in the pull request. Because these issues can't be reported as line comments, they are summarized here:

1. ![MINOR][MINOR] AncientFungus:LGEPR: Cppcheck cannot find all the include files (use --check-config for details) [rule](http://localhost:9000/coding_rules#rule_key=cppcheck%3AmissingInclude)
1. ![MINOR][MINOR] [RenderManager.cpp#L505](https://github.com/Chainsawkitten/LargeGameProjectEngine/blob/c413fbe2e2dd5eb0e63e2e9adfc47d4f784da82d/src/Engine/Manager/RenderManager.cpp#L505): The function 'GetAnimations' is never used. [rule](http://localhost:9000/coding_rules#rule_key=cppcheck%3AunusedFunction)
1. ![MINOR][MINOR] [RenderManager.cpp#L523](https://github.com/Chainsawkitten/LargeGameProjectEngine/blob/c413fbe2e2dd5eb0e63e2e9adfc47d4f784da82d/src/Engine/Manager/RenderManager.cpp#L523): The function 'GetDirectionalLights' is never used. [rule](http://localhost:9000/coding_rules#rule_key=cppcheck%3AunusedFunction)
1. ![MINOR][MINOR] [RenderManager.cpp#L542](https://github.com/Chainsawkitten/LargeGameProjectEngine/blob/c413fbe2e2dd5eb0e63e2e9adfc47d4f784da82d/src/Engine/Manager/RenderManager.cpp#L542): The function 'GetLenses' is never used. [rule](http://localhost:9000/coding_rules#rule_key=cppcheck%3AunusedFunction)
1. ![MINOR][MINOR] [RenderManager.cpp#L562](https://github.com/Chainsawkitten/LargeGameProjectEngine/blob/c413fbe2e2dd5eb0e63e2e9adfc47d4f784da82d/src/Engine/Manager/RenderManager.cpp#L562): The function 'GetMaterials' is never used. [rule](http://localhost:9000/coding_rules#rule_key=cppcheck%3AunusedFunction)
1. ![MINOR][MINOR] [RenderManager.cpp#L580](https://github.com/Chainsawkitten/LargeGameProjectEngine/blob/c413fbe2e2dd5eb0e63e2e9adfc47d4f784da82d/src/Engine/Manager/RenderManager.cpp#L580): The function 'GetMeshes' is never used. [rule](http://localhost:9000/coding_rules#rule_key=cppcheck%3AunusedFunction)
1. ![MINOR][MINOR] [RenderManager.cpp#L599](https://github.com/Chainsawkitten/LargeGameProjectEngine/blob/c413fbe2e2dd5eb0e63e2e9adfc47d4f784da82d/src/Engine/Manager/RenderManager.cpp#L599): The function 'GetPointLights' is never used. [rule](http://localhost:9000/coding_rules#rule_key=cppcheck%3AunusedFunction)
1. ![MINOR][MINOR] [RenderManager.cpp#L621](https://github.com/Chainsawkitten/LargeGameProjectEngine/blob/c413fbe2e2dd5eb0e63e2e9adfc47d4f784da82d/src/Engine/Manager/RenderManager.cpp#L621): The function 'GetSpotLights' is never used. [rule](http://localhost:9000/coding_rules#rule_key=cppcheck%3AunusedFunction)

[MINOR]: https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR'

**Comment by NinjaDanz3r** [comment]
SonarQube analysis reported 8 issues

Note: The following issues were found on lines that were not modified in the pull request. Because these issues can't be reported as line comments, they are summarized here:

1. ![MINOR][MINOR] AncientFungus:LGEPR: Cppcheck cannot find all the include files (use --check-config for details) [rule](http://localhost:9000/coding_rules#rule_key=cppcheck%3AmissingInclude)
1. ![MINOR][MINOR] [RenderManager.cpp#L505](https://github.com/Chainsawkitten/LargeGameProjectEngine/blob/6cec96b9e7b585e6d87a61e2f6b8d5baf538164e/src/Engine/Manager/RenderManager.cpp#L505): The function 'GetAnimations' is never used. [rule](http://localhost:9000/coding_rules#rule_key=cppcheck%3AunusedFunction)
1. ![MINOR][MINOR] [RenderManager.cpp#L523](https://github.com/Chainsawkitten/LargeGameProjectEngine/blob/6cec96b9e7b585e6d87a61e2f6b8d5baf538164e/src/Engine/Manager/RenderManager.cpp#L523): The function 'GetDirectionalLights' is never used. [rule](http://localhost:9000/coding_rules#rule_key=cppcheck%3AunusedFunction)
1. ![MINOR][MINOR] [RenderManager.cpp#L542](https://github.com/Chainsawkitten/LargeGameProjectEngine/blob/6cec96b9e7b585e6d87a61e2f6b8d5baf538164e/src/Engine/Manager/RenderManager.cpp#L542): The function 'GetLenses' is never used. [rule](http://localhost:9000/coding_rules#rule_key=cppcheck%3AunusedFunction)
1. ![MINOR][MINOR] [RenderManager.cpp#L562](https://github.com/Chainsawkitten/LargeGameProjectEngine/blob/6cec96b9e7b585e6d87a61e2f6b8d5baf538164e/src/Engine/Manager/RenderManager.cpp#L562): The function 'GetMaterials' is never used. [rule](http://localhost:9000/coding_rules#rule_key=cppcheck%3AunusedFunction)
1. ![MINOR][MINOR] [RenderManager.cpp#L580](https://github.com/Chainsawkitten/LargeGameProjectEngine/blob/6cec96b9e7b585e6d87a61e2f6b8d5baf538164e/src/Engine/Manager/RenderManager.cpp#L580): The function 'GetMeshes' is never used. [rule](http://localhost:9000/coding_rules#rule_key=cppcheck%3AunusedFunction)
1. ![MINOR][MINOR] [RenderManager.cpp#L599](https://github.com/Chainsawkitten/LargeGameProjectEngine/blob/6cec96b9e7b585e6d87a61e2f6b8d5baf538164e/src/Engine/Manager/RenderManager.cpp#L599): The function 'GetPointLights' is never used. [rule](http://localhost:9000/coding_rules#rule_key=cppcheck%3AunusedFunction)
1. ![MINOR][MINOR] [RenderManager.cpp#L621](https://github.com/Chainsawkitten/LargeGameProjectEngine/blob/6cec96b9e7b585e6d87a61e2f6b8d5baf538164e/src/Engine/Manager/RenderManager.cpp#L621): The function 'GetSpotLights' is never used. [rule](http://localhost:9000/coding_rules#rule_key=cppcheck%3AunusedFunction)

[MINOR]: https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR'

**Comment by NinjaDanz3r** [comment]
SonarQube analysis reported 8 issues

Note: The following issues were found on lines that were not modified in the pull request. Because these issues can't be reported as line comments, they are summarized here:

1. ![MINOR][MINOR] AncientFungus:LGEPR: Cppcheck cannot find all the include files (use --check-config for details) [rule](http://localhost:9000/coding_rules#rule_key=cppcheck%3AmissingInclude)
1. ![MINOR][MINOR] [RenderManager.cpp#L505](https://github.com/Chainsawkitten/LargeGameProjectEngine/blob/d7f252b67e4bb95c63e43c4a06ce2338499fe27f/src/Engine/Manager/RenderManager.cpp#L505): The function 'GetAnimations' is never used. [rule](http://localhost:9000/coding_rules#rule_key=cppcheck%3AunusedFunction)
1. ![MINOR][MINOR] [RenderManager.cpp#L523](https://github.com/Chainsawkitten/LargeGameProjectEngine/blob/d7f252b67e4bb95c63e43c4a06ce2338499fe27f/src/Engine/Manager/RenderManager.cpp#L523): The function 'GetDirectionalLights' is never used. [rule](http://localhost:9000/coding_rules#rule_key=cppcheck%3AunusedFunction)
1. ![MINOR][MINOR] [RenderManager.cpp#L542](https://github.com/Chainsawkitten/LargeGameProjectEngine/blob/d7f252b67e4bb95c63e43c4a06ce2338499fe27f/src/Engine/Manager/RenderManager.cpp#L542): The function 'GetLenses' is never used. [rule](http://localhost:9000/coding_rules#rule_key=cppcheck%3AunusedFunction)
1. ![MINOR][MINOR] [RenderManager.cpp#L562](https://github.com/Chainsawkitten/LargeGameProjectEngine/blob/d7f252b67e4bb95c63e43c4a06ce2338499fe27f/src/Engine/Manager/RenderManager.cpp#L562): The function 'GetMaterials' is never used. [rule](http://localhost:9000/coding_rules#rule_key=cppcheck%3AunusedFunction)
1. ![MINOR][MINOR] [RenderManager.cpp#L580](https://github.com/Chainsawkitten/LargeGameProjectEngine/blob/d7f252b67e4bb95c63e43c4a06ce2338499fe27f/src/Engine/Manager/RenderManager.cpp#L580): The function 'GetMeshes' is never used. [rule](http://localhost:9000/coding_rules#rule_key=cppcheck%3AunusedFunction)
1. ![MINOR][MINOR] [RenderManager.cpp#L599](https://github.com/Chainsawkitten/LargeGameProjectEngine/blob/d7f252b67e4bb95c63e43c4a06ce2338499fe27f/src/Engine/Manager/RenderManager.cpp#L599): The function 'GetPointLights' is never used. [rule](http://localhost:9000/coding_rules#rule_key=cppcheck%3AunusedFunction)
1. ![MINOR][MINOR] [RenderManager.cpp#L621](https://github.com/Chainsawkitten/LargeGameProjectEngine/blob/d7f252b67e4bb95c63e43c4a06ce2338499fe27f/src/Engine/Manager/RenderManager.cpp#L621): The function 'GetSpotLights' is never used. [rule](http://localhost:9000/coding_rules#rule_key=cppcheck%3AunusedFunction)

[MINOR]: https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR'

**Comment by NinjaDanz3r** [comment]
SonarQube analysis reported 8 issues

Note: The following issues were found on lines that were not modified in the pull request. Because these issues can't be reported as line comments, they are summarized here:

1. ![MINOR][MINOR] AncientFungus:LGEPR: Cppcheck cannot find all the include files (use --check-config for details) [rule](http://localhost:9000/coding_rules#rule_key=cppcheck%3AmissingInclude)
1. ![MINOR][MINOR] [RenderManager.cpp#L505](https://github.com/Chainsawkitten/LargeGameProjectEngine/blob/1e1748924af0db1cb9a38ba81080dada0e3cf8db/src/Engine/Manager/RenderManager.cpp#L505): The function 'GetAnimations' is never used. [rule](http://localhost:9000/coding_rules#rule_key=cppcheck%3AunusedFunction)
1. ![MINOR][MINOR] [RenderManager.cpp#L523](https://github.com/Chainsawkitten/LargeGameProjectEngine/blob/1e1748924af0db1cb9a38ba81080dada0e3cf8db/src/Engine/Manager/RenderManager.cpp#L523): The function 'GetDirectionalLights' is never used. [rule](http://localhost:9000/coding_rules#rule_key=cppcheck%3AunusedFunction)
1. ![MINOR][MINOR] [RenderManager.cpp#L542](https://github.com/Chainsawkitten/LargeGameProjectEngine/blob/1e1748924af0db1cb9a38ba81080dada0e3cf8db/src/Engine/Manager/RenderManager.cpp#L542): The function 'GetLenses' is never used. [rule](http://localhost:9000/coding_rules#rule_key=cppcheck%3AunusedFunction)
1. ![MINOR][MINOR] [RenderManager.cpp#L562](https://github.com/Chainsawkitten/LargeGameProjectEngine/blob/1e1748924af0db1cb9a38ba81080dada0e3cf8db/src/Engine/Manager/RenderManager.cpp#L562): The function 'GetMaterials' is never used. [rule](http://localhost:9000/coding_rules#rule_key=cppcheck%3AunusedFunction)
1. ![MINOR][MINOR] [RenderManager.cpp#L580](https://github.com/Chainsawkitten/LargeGameProjectEngine/blob/1e1748924af0db1cb9a38ba81080dada0e3cf8db/src/Engine/Manager/RenderManager.cpp#L580): The function 'GetMeshes' is never used. [rule](http://localhost:9000/coding_rules#rule_key=cppcheck%3AunusedFunction)
1. ![MINOR][MINOR] [RenderManager.cpp#L599](https://github.com/Chainsawkitten/LargeGameProjectEngine/blob/1e1748924af0db1cb9a38ba81080dada0e3cf8db/src/Engine/Manager/RenderManager.cpp#L599): The function 'GetPointLights' is never used. [rule](http://localhost:9000/coding_rules#rule_key=cppcheck%3AunusedFunction)
1. ![MINOR][MINOR] [RenderManager.cpp#L621](https://github.com/Chainsawkitten/LargeGameProjectEngine/blob/1e1748924af0db1cb9a38ba81080dada0e3cf8db/src/Engine/Manager/RenderManager.cpp#L621): The function 'GetSpotLights' is never used. [rule](http://localhost:9000/coding_rules#rule_key=cppcheck%3AunusedFunction)

[MINOR]: https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR'

**Comment by NinjaDanz3r** [comment]
SonarQube analysis reported 8 issues

Note: The following issues were found on lines that were not modified in the pull request. Because these issues can't be reported as line comments, they are summarized here:

1. ![MINOR][MINOR] AncientFungus:LGEPR: Cppcheck cannot find all the include files (use --check-config for details) [rule](http://localhost:9000/coding_rules#rule_key=cppcheck%3AmissingInclude)
1. ![MINOR][MINOR] [RenderManager.cpp#L505](https://github.com/Chainsawkitten/LargeGameProjectEngine/blob/d54701b50e2ea9761adc614534d68ddddb947864/src/Engine/Manager/RenderManager.cpp#L505): The function 'GetAnimations' is never used. [rule](http://localhost:9000/coding_rules#rule_key=cppcheck%3AunusedFunction)
1. ![MINOR][MINOR] [RenderManager.cpp#L523](https://github.com/Chainsawkitten/LargeGameProjectEngine/blob/d54701b50e2ea9761adc614534d68ddddb947864/src/Engine/Manager/RenderManager.cpp#L523): The function 'GetDirectionalLights' is never used. [rule](http://localhost:9000/coding_rules#rule_key=cppcheck%3AunusedFunction)
1. ![MINOR][MINOR] [RenderManager.cpp#L542](https://github.com/Chainsawkitten/LargeGameProjectEngine/blob/d54701b50e2ea9761adc614534d68ddddb947864/src/Engine/Manager/RenderManager.cpp#L542): The function 'GetLenses' is never used. [rule](http://localhost:9000/coding_rules#rule_key=cppcheck%3AunusedFunction)
1. ![MINOR][MINOR] [RenderManager.cpp#L562](https://github.com/Chainsawkitten/LargeGameProjectEngine/blob/d54701b50e2ea9761adc614534d68ddddb947864/src/Engine/Manager/RenderManager.cpp#L562): The function 'GetMaterials' is never used. [rule](http://localhost:9000/coding_rules#rule_key=cppcheck%3AunusedFunction)
1. ![MINOR][MINOR] [RenderManager.cpp#L580](https://github.com/Chainsawkitten/LargeGameProjectEngine/blob/d54701b50e2ea9761adc614534d68ddddb947864/src/Engine/Manager/RenderManager.cpp#L580): The function 'GetMeshes' is never used. [rule](http://localhost:9000/coding_rules#rule_key=cppcheck%3AunusedFunction)
1. ![MINOR][MINOR] [RenderManager.cpp#L599](https://github.com/Chainsawkitten/LargeGameProjectEngine/blob/d54701b50e2ea9761adc614534d68ddddb947864/src/Engine/Manager/RenderManager.cpp#L599): The function 'GetPointLights' is never used. [rule](http://localhost:9000/coding_rules#rule_key=cppcheck%3AunusedFunction)
1. ![MINOR][MINOR] [RenderManager.cpp#L621](https://github.com/Chainsawkitten/LargeGameProjectEngine/blob/d54701b50e2ea9761adc614534d68ddddb947864/src/Engine/Manager/RenderManager.cpp#L621): The function 'GetSpotLights' is never used. [rule](http://localhost:9000/coding_rules#rule_key=cppcheck%3AunusedFunction)

[MINOR]: https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR'

**Comment by NinjaDanz3r** [comment]
SonarQube analysis reported 8 issues

Note: The following issues were found on lines that were not modified in the pull request. Because these issues can't be reported as line comments, they are summarized here:

1. ![MINOR][MINOR] AncientFungus:LGEPR: Cppcheck cannot find all the include files (use --check-config for details) [rule](http://localhost:9000/coding_rules#rule_key=cppcheck%3AmissingInclude)
1. ![MINOR][MINOR] [RenderManager.cpp#L505](https://github.com/Chainsawkitten/LargeGameProjectEngine/blob/ee0e8308f1820cba6a6aa8a6ce53ca6b5cd4b282/src/Engine/Manager/RenderManager.cpp#L505): The function 'GetAnimations' is never used. [rule](http://localhost:9000/coding_rules#rule_key=cppcheck%3AunusedFunction)
1. ![MINOR][MINOR] [RenderManager.cpp#L523](https://github.com/Chainsawkitten/LargeGameProjectEngine/blob/ee0e8308f1820cba6a6aa8a6ce53ca6b5cd4b282/src/Engine/Manager/RenderManager.cpp#L523): The function 'GetDirectionalLights' is never used. [rule](http://localhost:9000/coding_rules#rule_key=cppcheck%3AunusedFunction)
1. ![MINOR][MINOR] [RenderManager.cpp#L542](https://github.com/Chainsawkitten/LargeGameProjectEngine/blob/ee0e8308f1820cba6a6aa8a6ce53ca6b5cd4b282/src/Engine/Manager/RenderManager.cpp#L542): The function 'GetLenses' is never used. [rule](http://localhost:9000/coding_rules#rule_key=cppcheck%3AunusedFunction)
1. ![MINOR][MINOR] [RenderManager.cpp#L562](https://github.com/Chainsawkitten/LargeGameProjectEngine/blob/ee0e8308f1820cba6a6aa8a6ce53ca6b5cd4b282/src/Engine/Manager/RenderManager.cpp#L562): The function 'GetMaterials' is never used. [rule](http://localhost:9000/coding_rules#rule_key=cppcheck%3AunusedFunction)
1. ![MINOR][MINOR] [RenderManager.cpp#L580](https://github.com/Chainsawkitten/LargeGameProjectEngine/blob/ee0e8308f1820cba6a6aa8a6ce53ca6b5cd4b282/src/Engine/Manager/RenderManager.cpp#L580): The function 'GetMeshes' is never used. [rule](http://localhost:9000/coding_rules#rule_key=cppcheck%3AunusedFunction)
1. ![MINOR][MINOR] [RenderManager.cpp#L599](https://github.com/Chainsawkitten/LargeGameProjectEngine/blob/ee0e8308f1820cba6a6aa8a6ce53ca6b5cd4b282/src/Engine/Manager/RenderManager.cpp#L599): The function 'GetPointLights' is never used. [rule](http://localhost:9000/coding_rules#rule_key=cppcheck%3AunusedFunction)
1. ![MINOR][MINOR] [RenderManager.cpp#L621](https://github.com/Chainsawkitten/LargeGameProjectEngine/blob/ee0e8308f1820cba6a6aa8a6ce53ca6b5cd4b282/src/Engine/Manager/RenderManager.cpp#L621): The function 'GetSpotLights' is never used. [rule](http://localhost:9000/coding_rules#rule_key=cppcheck%3AunusedFunction)

[MINOR]: https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR'

**Pull request merged**